python/kfserving/docs/KFServingClient.md

```diff
@@ -16,11 +16,66 @@ The APIs for KFServingClient are as following:
 
 Class | Method |  Description
 ------------ | ------------- | -------------
+KFServingClient | [create_creds](#create_creds) | Create Credentials|
 KFServingClient | [create](#create) | Create KFService|
 KFServingClient | [get](#get)    | Get the specified KFService|
 KFServingClient | [patch](#patch)  | Patch the specified KFService|
 KFServingClient | [delete](#delete) | Delete the specified KFService |
 
+## create_creds
+> create_creds(self, platform, namespace=None, **kwargs)
+
+Create `Secret` and `Service Account` for AWS and GCP accordingly to credentials information. Once the `Service Account` ceated, user can configure the `Service Account` in the [V1alpha1ModelSpec](V1alpha1ModelSpec.md) for kfserving.
+
+The API returns name of created `Service Account`.
+
+### Example
+
+Example for creating AWS credentials.
+```python
+KFServingClient
+from kfserving import KFServingClient
+
+KFServing = KFServingClient()
+KFServing.create_creds(platform='AWS',
+                          namespace='kubeflow',
+                          AWS_ACCESS_KEY_ID='MWYyZDFlMmU2N2Rm',
```

Comment:
Should we read these from boto or something? I feel like exposing credential directly is a really bad idea.

Either we should read from the local aws application credentials or we should call IAM

Reply:
Are they AWS creds or S3 creds? These are credentials for storage, so it should be more like
KFServing.create_creds(storage_type='s3')
KFServing.create_creds(storage_type='gcs')

Reply:
Updated to read the credentials from `~/.aws/credentials` file. Thanks.